std/random.d

```diff
@@ -2182,8 +2182,7 @@ do
         immutable T u = (rng.front - rng.min) * factor;
         rng.popFront();
 
-        import core.stdc.limits : CHAR_BIT;  // CHAR_BIT is always 8
```

Comment:
What if on some platform, someday, it isn't 8?

Reply:
Then `T.sizeof` wouldn't be in 8-bit bytes anymore, and all hell breaks loose I guess.